sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuShuffledHashJoinBase.scala

```diff
@@ -68,37 +69,25 @@ abstract class GpuShuffledHashJoinBase(
     val joinTime = gpuLongMetric(JOIN_TIME)
     val filterTime = gpuLongMetric(FILTER_TIME)
     val joinOutputRows = gpuLongMetric(JOIN_OUTPUT_ROWS)
-
-    val boundCondition = condition.map(GpuBindReferences.bindReference(_, output))
+    val spillCallback = GpuMetric.makeSpillCallback(allMetrics)
+    val localBuildOutput: Seq[Attribute] = buildPlan.output
```

Comment:
Similar comment re: caching the `targetSize` in a local to help avoid serializing too much